lms/envs/common.py

```diff
@@ -1426,7 +1426,6 @@ def _make_locale_paths(settings):
     sorted(rooted_glob(PROJECT_ROOT / 'static', 'js/dashboard/**/*.js'))
 )
 discussion_js = (
-    rooted_glob(COMMON_ROOT / 'static', 'common/js/discussion/mathjax_include.js') +
```

Comment:
Just to add clarity, why you are removing it?

Reply:
On line 1432, the rooted_glob call is also including mathjax_include.js. That resulted in the multiple loading of the file. That's why it was unnecessary to add it standalone.

Reply:
thank you.